build-logic/profiling/src/main/kotlin/gradlebuild.buildscan.gradle.kts

```diff
@@ -116,13 +117,15 @@ fun isExpectedCompileCacheMiss() =
 // 3. buildScanPerformance test, which doesn't depend on compileAll
     // 4. buildScanPerformance test, which doesn't depend on compileAll
     isInBuild(
-        "Gradle_Check_CompileAll",
-        "Enterprise_Master_Components_GradleBuildScansPlugin_Performance_PerformanceLinux",
-        "Enterprise_Release_Components_BuildScansPlugin_Performance_PerformanceLinux",
-        "Gradle_Check_Gradleception"
+        "Check_CompileAll",
+        "Check_CompileAll",
+        "Components_GradleBuildScansPlugin_Performance_PerformanceLinux",
```

Comment:
It seems like the build type ids of these things have changed to:
- `Enterprise_Master_Component_GradlePlugin_Performance_PerformanceLatestMaster` and
- `Enterprise_Master_Component_GradlePlugin_Performance_PerformanceLatestReleased`

Reply:
Thanks! I copied from TeamCity.